src/deployments/kubernetes/app-secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: app-secret
+type: Opaque
+data:
+    postgres-password: dG9wc2VjdXJl
```

Comment:
Is this essentially the equivalent of a `.env` file? If so, would it make sense to rename this as `.example` and force people to create their own?

Would help allay some concern about having a hard-coded password in the repository.

Reply:
We can say that this is a .env file for secrets only, see https://kubernetes.io/docs/concepts/configuration/secret/ for more info.
That value is encrypted and will be set unencrypted in the cluster, this is, in a way a .example file.

In Meadowlark, we talked about keeping it for demo purposes, but not sure if we should remove the file or add a warning in the README about the file. https://github.com/Ed-Fi-Exchange-OSS/Meadowlark/pull/337#discussion_r1446380880

Reply:
I think its ok if someone has to create this file, and its ok if we provide an example. I just want to lower the risk that someone accidentally uses this in a more serious environment without thought. Maybe we add `app-secret.yaml` to the `.gitignore` file.